plugins/scaffolder-backend-module-confluence-to-markdowwn/package.json

```
@@ -0,0 +1,52 @@
+{
+  "name": "@backstage/plugin-scaffolder-backend-module-confluence-to-markdown",
+  "description": "The confluence-to-markdown module for @backstage/plugin-scaffolder-backend",
+  "version": "0.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.cjs.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "backend-plugin-module"
+  },
+  "scripts": {
+    "start": "backstage-cli package start",
+    "build": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test",
+    "clean": "backstage-cli package clean",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack",
+    "help": "backstage-cli help"
+  },
+  "dependencies": {
+    "@backstage/backend-common": "workspace:^",
+    "@backstage/config": "workspace:^",
+    "@backstage/errors": "workspace:^",
+    "@backstage/integration": "workspace:^",
+    "@backstage/plugin-scaffolder-backend": "workspace:^",
+    "@backstage/plugin-scaffolder-node": "workspace:^",
+    "@backstage/types": "workspace:^",
+    "fs-extra": "^11.1.0",
+    "git-url-parse": "^13.1.0",
+    "jest": "^29.5.0",
+    "node-fetch": "2.6.7",
+    "node-html-markdown": "^1.3.0",
+    "ts-jest": "^29.0.5",
+    "yaml": "^2.2.1"
```

```suggestion
    "yaml": "^2.0.0"
```
Lets make this ^2.0.0 to conform with most existing deps in the repo